app/views/layouts/forever_style_guide/application.html.erb

```diff
@@ -14,7 +14,11 @@
   <%= csrf_meta_tags %>
 </head>
 <body class="style-guide" data-spy="scroll" data-target=".style-guide-scrollspy">
-  <%= render partial: "layouts/forever_style_guide/navigation" %>
+
+  <% if !content_for?(:hide_nav) %>
```

Comment:
Cool. This was definitely hacky and I'm glad to make it better. I was a little unclear about how the controllers work in the style guide, but I'll take a closer look and get this updated.